www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md

```diff
@@ -8,12 +8,12 @@ loading configuration: none (standalone session)
 Session type: standalone
 Loading.............done.
 
-Welcome to the chef-shell 15.8.23
+Welcome to the chef-shell 15.10.12
```

Comment:
Obviously not a blocker, but you don't need an indefinite article:

```suggestion
Welcome to chef-shell 15.10.12
```

Reply:
That text is dated now, but it's the actual text from the command

Reply:
I assumed--it just stands out.